src/documents/IoCSS.md

```diff
@@ -47,74 +52,78 @@ fluid.uiOptions(".my-uio-container", {
 ## IoCSS Selectors ##
 
 Component matching rules:
 
 | Form | Description |
 |------|-------------|
-|*|matches any component - universal selector|
-|E|matches any component holding a context name of E - special support for the string "that" as with standard IoC context matching|
-|E#myid|matches any component with a context name of E with id equal to myid (of no use to developers since component ids cannot be predicted)|
+|`*`|matches any component - universal selector|
+|`/`|matches the **global root component** of the entire component tree - may only be used in the head position of an IoCSS selector|
+|`E` or `&E`|matches any component holding a context name of `E` - the `&` character may be omitted for the first context in a group. There is special support for the context `that` as with standard IoC context matching|
+|`#myid`|matches the component with id equal to myid (of no use to developers since component ids cannot be predicted)|
 
-Descendent rules
+Descendant rules
 
 | Form | Description |
 |------|-------------|
-|E F|Matches any F component that is a descendant of an E component|
-|E > F|Matches any F component that is a direct child of an E component|
+|`E F`|Matches any F component that is a descendant of an `E` component|
+|`E > F`|Matches any F component that is a direct child of an `E` component|
 
-## Example: source ##
-```javascript
-fluid.defaults("fluid.tests.uploader", {
-    gradeNames: ["fluid.littleComponent", "autoInit"],
-    components: {
-        uploaderContext: {
-            type: "fluid.progressiveCheckerForComponent",
-            options: {componentName: "fluid.tests.uploader"}
-        },
-        uploaderImpl: {
-            type: "fluid.tests.uploaderImpl"
-        }
-    },
-    distributeOptions: [{
-        target: "{that > uploaderImpl}.options" // Target a directly nested component matching the context "uploaderImpl"
-        source: "{that}.options",               // Distribute ALL of our options there, except exclusions:
-        exclusions: ["components.uploaderContext", "components.uploaderImpl"], // options targetted directly at these subcomponents are left undisturbed in place
+## Simple Example: `record` for grade distribution ##
 
-    }],
-    progressiveCheckerOptions: {
-        checks: [{
-            feature: "{fluid.test}",
-            contextName: "fluid.uploader.html5"
-        }]
+This is the form taken by the most commonly seen and straightforward use of `distributeOptions` - the contribution of an additional grade to
+one or more subcomponents of the current component:
+
+```javascript
+fluid.defaults("kettle.use.session.io", {
+    gradeNames: ["fluid.component"],
+    distributeOptions: {
+        record: "kettle.sessionManager.io",
+        target: "{that sessionManager}.options.gradeNames"
     }
 });
 ```
 
-## Example: record ##
+The `sessionManager`, wherever it is in the set of child components, will be granted the additional grade `kettle.sessionManager.io`.
```

Comment:
Given the refactoring of Kettle, is the example somewhat misleading now? Should we use a different example?